### PR TITLE
Fixes and improvements of Ledger Integration

### PR DESCRIPTION
**Pull request by sarveshwar-singh**
Fixes and improvements of Ledger Integration

### CODE REVIEW

File: app/i18n/locales/ja-JP.json [hunk 1]
```diff
@@ -377,36 +377,5 @@
...
-  "wallet.trezor.dialog.step.connect.label": "接続",
-  "wallet.trezor.dialog.step.save.error.101": "保存に失敗しました。インターネット接続を確認して再試行してください。",
-  "wallet.trezor.dialog.step.save.label": "保存",
-  "wallet.trezor.dialog.step.save.walletName.hint": "ウォレット名を入力",
-  "wallet.trezor.dialog.step.save.walletName.info": "Trezorデバイス名を読み出しました。このまま使用するか、任意の名前を入力してください。",
-  "wallet.trezor.dialog.step.save.walletName.label": "ウォレット名",
-  "wallet.trezor.dialog.title.label": "Trezorハードウェアウォレットに接続",
-  "wallet.trezor.error.101": "Trezor.ioへの接続に失敗しました。インターネット接続を確認して再試行してください。",
-  "wallet.trezor.error.102": "アクセス許可が得られませんでした。再試行してください。",
-  "wallet.trezor.error.103": "キャンセルされました。再試行してください。"
```
Comment (vantuz-subhuman) [inline app/i18n/locales/ja-JP.json]: Why this big chunk is just removed? O_o

**Comment by nicarq** [comment]
The indonesian file name is wrong. It should follow the format de-DE, en-US, etc. This is *not* that someone did something wrong, but when it’s added from Crowdin , it’s added like that

File: app/i18n/locales/ja-JP.json (hunk 1 above)
Comment (sarveshwar-singh) [reply app/i18n/locales/ja-JP.json]: its renamed from `wallet.trezor.dialog` -> `wallet.connect.trezor.dialog`

Comment (vantuz-subhuman) [reply app/i18n/locales/ja-JP.json]: Understood, thank you!

**Pull request merged**